Week3/homework/step2-6.js

```diff
@@ -4,20 +4,30 @@ const arr2d = [[1, 2], [3, 4], [5, 6]];
 const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
 function flattenArray2d(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+	let arr2Flat = [].concat.apply([], arr2d);
```

Comment:
Not sure this is the best approach. Have you checked out .flat() method?

Reply:
Yeap, didn´t know how to use it

Reply:
Can you replace with .flat()?

Reply:
done, but not working